tests/test_memcache.py

```diff
@@ -1,10 +1,10 @@
 from __future__ import print_function
 
-import unittest
-
+import mock
 import six
+import unittest
```

Comment:
This import shouldn't be resorted. Standard library imports are separated from third-party imports.

Reply:
That's not true, locally it failed flake8 due to the imports needing to be alphabetical for lsort.

Reply:
Sorry, I didn't know that.

Reply:
Yeah, I only included it in the test-requirements.txt for users of python 2.7